src/tests/control/test_auth.py

```diff
@@ -787,7 +787,8 @@ def test_update_session_activity(self):
         assert self.client.session['pretix_auth_last_used'] > t1
 
     def test_pinned_user_agent(self):
-        self.client.defaults['HTTP_USER_AGENT'] = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36'
+        self.client.defaults[
+            'HTTP_USER_AGENT'] = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36'
```

Comment:
I'm pretty sure this was your editor, not you, but it looks less-readbale for me. If we want to break this line, maybe like this?

```suggestion
        self.client.defaults['HTTP_USER_AGENT'] = (
            'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) '
            'Chrome/64.0.3282.140 Safari/537.36'
        )
```

Reply:
I used 3 lines, but it does look better, and yes, my editor was at fault. It "corrected" it.